Drivers/Src/AS8510.c

```diff
@@ -33,12 +33,48 @@
 #define AS8510_DEC_R2_V      0x46
 #define AS8510_FIR_CTL_REG_V 0x47
 
+static OS_SEM AmperesIO_Sem;
+static bsp_os_t spi_os;
+
+#ifdef RTOS
+void Amperes_Pend(){
```

Comment:
This semaphore is used by the SPI3 ISR to signal when there is an interrupt (SPI message received from AS8510). I put it here because `BPS_SPI_Init()` uses this as a parameter. I suppose it might make more sense to put this in Amps.c and pass this as a parameter `AS8510_Init()`